extruct/tool.py

```diff
@@ -14,8 +14,8 @@ def metadata_from_url(url, syntaxes=SYNTAXES, uniform=False,
         resp.raise_for_status()
     except requests.exceptions.HTTPError:
         return result
-    result.update(extruct.extract(resp.content, 
-                                  url=url, 
+    result.update(extruct.extract(resp.content,
+                                  base_url=url,  # FIXME: use base url
```

Comment:
One more reason to deprecate the tool - I was going to suggest using get_base_url from w3lib, but it seems this would require more extra code to handle encodings correctly (and we are already throwing encoding information away here by not using headers).

Reply:
Yeah, actually I started adding get_base_url, then realized encodings need to be handled, etc., removed that, and just added FIXME comment instead :)